docs/en/Extensions.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: global
+group: Features
+title: Under Storage Extensions
+priority: 7
```

Comment:
@gpang  do we still use `priority` ?

Reply:
If we remove this, it will be alphabetical in the list. Did we already remove the priorities on others?